build/Makefile.objecthash

```diff
@@ -0,0 +1,4 @@
+all: test
+
+test:
+	$(MAKE) c CPPFLAGS="-I $(INSTALL_DIR)/include" LDFLAGS="-L $(INSTALL_DIR)/lib"
```

Comment:
Is this even necessary? Since you link `third_party/objecthash/objecthash.c` straight into the makefile?